xmlseclibs.php

```diff
@@ -40,6 +40,10 @@
  * @version   2.1.0-dev
  */
 
+if (!function_exists('random_bytes')) {
+    die('xmlseclibs requires https://github.com/paragonie/random_compat for PHP <7.0');
+}
```

Comment:
Keeping this file for now. I'd argue to get rid of it entirely though with a proper version bump.